docs/proposals/kubeedgeadm-scope.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Kubeedgeadm kubeEdge installer scope (Issue 324)
+status: Pending
+authors:
+    - "@samy2019"
+approvers:
+  - "@m1093782566"
+  - "@rohitsardesai83"
+  - "@sids-b"
+creation-date: 2019-04-11
+last-updated: 2019-04-12
+---
+
+## Motivation
+Many users shared their feedback that kubeEdge installation is too complicated and it may prevent people from trying kubeEdge.
+There should a simplified way to have **Getting Started with KubeEdge**, so that user can concentrate more on using it instantly, rather than
+getting entangled in the installation steps.
+
+## Proposal
+KubeEdge shall have simple commands and steps to bring up both cloud and edge components.
+The user experience in **Getting Started with KubeEdge** will be seamless.
+Hence proposing the following commands for KubeEdge installation process.
+
+For cloud, commands shall be:
+
+- `kubeedge cloud init`
+- `kubeedge cloud reset`
+
+For edge, commands shall be:
+
+- `kubeedge node init`
+- `kubeedge node reset`
+- `kubeedge node join`
+
+**NOTE:**
+`node` key is used for edge component in the command, for superficial reasons. Because `kubeedge edge init` had `edge` used twice and didn't sound nice.
+
+**ASSUMPTIONS**
```

Comment:
kubeadm installs a specific version of k8s : 
https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#support-timeframes.

Do we plan to have our kubeedge install utility to download a specific version of kubeedge release ( like say v0.2 ? ) ?
Since we are targeting users of kubeedge , we probably won't expect them to clone the kubeedge repo .

Reply:
Good point. Will add version support in kubeedge init command

Reply:
Added